vmas/simulator/sensors.py

```diff
@@ -123,9 +136,10 @@ def render(self, env_index: int = 0) -> "List[Geom]":
                 xform.set_translation(*self.agent.state.pos[env_index])
                 xform.set_rotation(angle)
                 ray.add_attr(xform)
+                ray.set_color(*Color.BLACK.value, alpha=self.alpha)
```

Comment:
```suggestion
                ray.set_color(r=0, g=0, b=0, alpha=self.alpha)
```
The default black is not 0,0,0, so to keep the color it was before we have to use this